descriptors/ModuleDescriptor-template.json

```diff
@@ -795,6 +795,30 @@
           "delay": "1"
         }
       ]
+    },
```

Comment:
We need a dependency on pub-sub interfaces. Also, please add pub/sub event publish permission to the endpoints' permission lists (i.e. check-in endpoint, check-out, etc.)

Reply:
Added